artemis-core/src/main/java/org/jnosql/artemis/DatabaseType.java

```diff
@@ -24,5 +24,18 @@
  * Defines the database type supported on Artemis.
  */
 public enum DatabaseType {
-    DOCUMENT, COLUMN, KEY_VALUE, GRAPH;
+    DOCUMENT("DocumentCollectionManager"),
```

Comment:
@danielsoro could  you fixe this point: https://github.com/JNOSQL/artemis/issues/15

Thanks